lib/adapters/bitfinex-adapter.js

```diff
@@ -45,18 +45,10 @@ class BitfinexAdapter {
    * @returns {Promise<{authToken: string, expiresAt: number}>}
    */
   async refreshToken () {
-    try {
```

Comment:
@tarcisiozf could you explain why we remove error message formatting here?

Reply:
this used to be a user-facing error, now the server intercepts and handles it in a better fashion, also, the message did not corresponded with the labels api management UI. ref: https://github.com/bitfinexcom/bfx-hf-server/blob/master/lib/ws_servers/api/algos/algo_worker.js#L74